modules/file-integrity-node-status-failure.adoc

```diff
@@ -112,3 +108,5 @@ File: /hostroot/etc/resolv.conf
 
 Events:  <none>
 ----
+
+Due to the config map data size limit, AIDE logs over 1mb are added to the failure config map as a base64-encoded gzip archive.
```

Comment:
Better to add more details about how to extract the logs and when need to extract the logs, like below:
 In this case, you will want to pipe the output of the above command to `base64 -d | gunzip`. Compressed logs are indicated by the presense of a file-integrity.openshift.io/compressed annotation key in the configMap.

Reply:
+1